stablehlo/dialect/VhloOps.td

```diff
@@ -869,95 +869,95 @@ def VHLO_OptimizationBarrierOpV1 : VHLO_Op<"optimization_barrier"> {
-  let arguments = (ins VHLO_QuantizedIntTensor:$operand);
-  let results = (outs TensorOf<[F32, BF16]>:$result);
+  let arguments = (ins VHLO_AnyType:$operand);
+  let results = (outs VHLO_AnyType:$result);
 }
 
 def VHLO_ReducePrecisionOpV1 : VHLO_Op<"reduce_precision"> {
   let arguments = (ins
-    VHLO_FpTensor:$operand,
-    I32Attr:$exponent_bits,
-    I32Attr:$mantissa_bits
+    VHLO_AnyType:$operand,
+    VHLO_AnyAttr:$exponent_bits,
+    VHLO_AnyAttr:$mantissa_bits
   );
-  let results = (outs VHLO_FpTensor:$output);
+  let results = (outs VHLO_AnyType:$output);
 }
 
 def VHLO_RealDynamicSliceOpV1 : VHLO_Op<"real_dynamic_slice"> {
   let arguments = (ins
-    VHLO_Tensor:$operand,
-    VHLO_DimensionTensor:$start_indices,
-    VHLO_DimensionTensor:$limit_indices,
-    VHLO_DimensionTensor:$strides
+    VHLO_AnyType:$operand,
+    VHLO_AnyType:$start_indices,
+    VHLO_AnyType:$limit_indices,
+    VHLO_AnyType:$strides
   );
-  let results = (outs VHLO_Tensor:$result);
+  let results = (outs VHLO_AnyType:$result);
 }
 
 def VHLO_DynamicPadOpV1 : VHLO_Op<"dynamic_pad"> {
   let arguments = (ins
-    VHLO_Tensor:$operand,
-    VHLO_Tensor:$padding_value,
-    VHLO_DimensionTensor:$edge_padding_low,
-    VHLO_DimensionTensor:$edge_padding_high,
-    VHLO_DimensionTensor:$interior_padding
+    VHLO_AnyType:$operand,
+    VHLO_AnyType:$padding_value,
+    VHLO_AnyType:$edge_padding_low,
+    VHLO_AnyType:$edge_padding_high,
+    VHLO_AnyType:$interior_padding
   );
-  let results = (outs VHLO_Tensor:$result);
+  let results = (outs VHLO_AnyType:$result);
 }
 
 def VHLO_DynamicGatherOpV1 : VHLO_Op<"dynamic_gather"> {
   let arguments = (ins
-    VHLO_Tensor:$operand,
-    VHLO_IntTensor:$start_indices,
-    VHLO_IntTensor:$slice_sizes,
-    VHLO_GatherDimensionNumbers:$dimension_numbers,
+    VHLO_AnyType:$operand,
+    VHLO_AnyType:$start_indices,
+    VHLO_AnyType:$slice_sizes,
+    VHLO_AnyAttr:$dimension_numbers,
     DefaultValuedOptionalAttr<BoolAttr, "false">:$indices_are_sorted
   );
-  let results = (outs VHLO_Tensor);
+  let results = (outs VHLO_AnyType);
 }
 
 def VHLO_DynamicConvOpV1 : VHLO_Op<"dynamic_conv"> {
   let arguments = !con(
     (ins
-       VHLO_Tensor:$lhs,
-       VHLO_Tensor:$rhs,
-       VHLO_Tensor:$d_padding),
+       VHLO_AnyType:$lhs,
+       VHLO_AnyType:$rhs,
+       VHLO_AnyType:$d_padding),
     VHLO_ConvolutionAttributes.attributes);
-  let results = (outs VHLO_Tensor);
+  let results = (outs VHLO_AnyType);
 }
 
 def VHLO_ComputeReshapeShapeOpV1 : VHLO_Op<"compute_reshape_shape"> {
-  let arguments = (ins Index:$num_elements, 1DTensorOf<[AnyInteger, Index]>:$dynamic_shape);
-  let results = (outs 1DTensorOf<[AnyInteger, Index]>:$result);
+  let arguments = (ins VHLO_AnyType:$num_elements, VHLO_AnyType:$dynamic_shape);
+  let results = (outs VHLO_AnyType:$result);
 }
 
 def VHLO_CstrReshapableOpV1 : VHLO_Op<"cstr_reshapable"> {
   let results = (outs Shape_WitnessType:$result);
```

Comment:
Not sure I follow the reasoning for why not change this to `VHLO_AnyType`.

Reply:
Agree. I was working on this in parallel to forking types and had already forked builtin tensor.

I think it would be good after we have all this forked to update the definition of `VHLO_AnyType` to be any of the _VHLO_ Types, as opposed to any type at all.

Reply:
Sounds like we no longer need `include "mlir/Dialect/Shape/IR/ShapeBase.td"` in the updated version of VhloOps.td?

Reply:
removed.